rxbindings/src/test/java/com/amplifyframework/rx/RxAuthBindingTest.java

```diff
@@ -796,6 +811,178 @@ public void testFetchDevices() throws InterruptedException {
                 .assertValue(expected);
     }
 
+    /**
+     * Tests that a successful request to fetch user attributes will propagate a completion
+     * back through the binding.
+     * @throws InterruptedException  If test observer is interrupted while awaiting terminal event
+     */
+    @Test
+    public void testFetchUserAttributes() throws InterruptedException {
+        // Arrange an invocation of the success Action
+        List<AuthUserAttribute> expected = Collections.singletonList(
+                new AuthUserAttribute(new AuthUserAttributeKey(ATTRIBUTE_KEY), ATTRIBUTE_VAL)
```

Comment:
Can this just be `AuthUserAttributeKey.email()`?

Reply:
Yes. But I was thinking about using the existing fields defined at the beginning.